src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/MathFunctionsIT.java

```diff
@@ -138,6 +138,42 @@ public void sinh() throws IOException {
         assertThat(sinh, equalTo(Math.sinh(Math.PI)));
     }
 
+    @Test
+    public void power() throws IOException {
+        SearchHit[] hits = query(
+                "SELECT POWER(PI(), 2) AS power"
```

Comment:
Please use these new functions in `WHERE` condition too because the code path may be a little different.

Reply:
@dai-chen Done. Could you take a look at the condition I've added, is that good or do I need to add more cases? Thanks!

Reply:
Thanks! Could you make function call in `WHERE` different from that in `SELECT`? Then you can add another `assertTrue` for script generated for `WHERE`. ex `SELECT POWER(age, 2) FROM bank WHERE POWER(balance, 3) > 0`. If possible, could you also do this for other test cases in IT?

Reply:
"other test cases" means only new tests you added.

Reply:
@dai-chen Done. Thanks!

Reply:
Thanks!